cmd/tendermint/commands/reindex_event.go

```diff
@@ -143,38 +143,38 @@ func eventReIndex(cmd *cobra.Command, args eventReIndexArgs) error {
 
 	fmt.Println("start re-indexing events:")
 	defer bar.Finish()
-	for i := args.startHeight; i <= args.endHeight; i++ {
+	for height := args.startHeight; height <= args.endHeight; height++ {
```

Comment:
[minor] This is not something one does when backporting.
If there are further patches to backport on top of this one, this will create unnecessary conflicts